lessons/pydata/clt/index.ipynb

```diff
@@ -0,0 +1,690 @@
+{
```

Comment:
<p>Popis příkladu s exponenciálním rozdělením je nepřesný až zavádějící. U tohohle rozdělení je pravděpodobnost objevení se poruchy u jednoho konkrétního výrobku v každém okamžiku konstantní (jen se zmenšuje populace, ze které se vybírá). Počet už porouchaných (~1 - exp(-C.t)) narůstá stále pomaleji, ne stále rychleji.</p><p> </p><p>Otázka je, jestli nějak nepopsat rozdíl mezi pdf a cdf. A případně i mezi diskrétním a spojitým rozdělením.</p><p> </p><p> </p><p>Drobnosti:</p><p> </p><p>&gt; Reprezentativnost dat a Centrální limitní věta</p><p> </p><p>Reprezentativnost dat a centrální limitní věta</p><p> </p><p>&gt; jak využít Centrálně limitní větu k tomu</p><p> </p><p>centrální limitní větu</p><p> </p><p>&gt; abychom se tomu vyvarovali.</p><p> </p><p>...toho...</p><p> </p><p>&gt; které se liší jen v tom zda lék užívají či nikoli</p><p> </p><p>...jen v tom, zda...</p><p> </p><p>&gt; předplatné v posilovně a tak za jejich uzdravením</p><p> </p><p>...posilovně, a tak</p><p> </p><p>&gt; popsaný problém z nenáhodným</p><p> </p><p>...s nenáhodným</p><p> </p><p>&gt; že učitel vydělává více než je určitá hodnota</p><p> </p><p>...více, než je</p><p> </p><p>- daná proměnná je skutečně distribuována tímto způsobem může být důležité pro některé konkrétní statistické výpočty.</p><p> </p><p>...způsobem, může</p><p> </p><p>&gt; Pro představu je tady jejich přibližné schéma:</p><p> </p><p>Následující schéma ukazuje celou řadu různých rozdělení a vztahů mezi nimi:</p><p> </p><p>&gt; znalost centrálně limitní vět</p><p> </p><p>centrální limitní věty</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ani bych neřekla, že je příklad nepřesný, myslím, že je dobře vysvětlen pro běžného uživatele. Životnost určitého výrobku je běžně používaným příkladem pro vysvětlení exp. rozdělení pravděpodobností. Berme v úvahu, že by to mělo být uživatelsky přívětivé a vysvětlovat to na vypočtu hustoty pravděpodobnosti dle poissonovského náhodného jevu by bylo nad rámec hodiny, to ať si každý dohledá sám.

Reply:
<p>Klidně se hustota pravděpodobnosti zmiňovat nemusí, ale stejně by se u pravděpodobnosti poruchy hodilo být konkrétnější v tom, zda jde o to, "zda se teď objeví porucha", nebo o to, že "je výrobek již porouchaný". A pravděpodobnost poruchy neroste "stále rychleji" ani v jedné interpretaci, tak je potřeba opravit aspoň toto.</p>

Reply:
To je pravda v tomto případě by mělo jít o poruchu nahodilou ne vlivem stárnutí materiálu.